### PR TITLE
MacOS Qt5 Guide Update

### PR DESCRIPTION
**Pull request by danston**
It was reported by the user that the current guide to compile CGAL examples with Qt5 on MacOS with brew is outdated:

-DQt5_DIR=/usr/local/opt/**qt**/lib/cmake/Qt5

and he could not succeed by following it.

I have checked that and indeed, after the Qt6 was introduced, the brew default Qt installation is Qt6, which is always linked to the `qt` folder whereas Qt5 is now linked to the `qt5` folder. I have updated the description to better explain how to link against Qt5.

## Release Management

* Affected package(s): `Documentation`
* Issue(s) solved (if any): reported in the mailing list 
* Feature/Small Feature (if any): doc bug fix
* Link to compiled documentation (obligatory for small feature): [docs](https://cgal.geometryfactory.com/~danston/Manual-qt5/usage.html#usage_configuring)
* License and copyright ownership: no change

### CODE REVIEW

File: Documentation/doc/Documentation/Usage.txt [hunk 1]
```diff
@@ -166,12 +166,17 @@ if no debugging is intended. Users should thus run:
     cd CGAL-\cgalReleaseNumber/examples/Triangulation_2
     cmake -DCGAL_DIR=$HOME/CGAL-\cgalReleaseNumber -DCMAKE_BUILD_TYPE=Release . # we are here using a release tarball
 
-Note that the package Qt on brew is "keg-only", which means it cannot be "linked" with brew.
-You will have to specify the Qt5_DIR by hand to cmake, using something like
+After the Qt6 is released, the package Qt5 on brew is "keg-only", which means it is not "linked" with brew.
```
Comment (sloriot) [inline Documentation/doc/Documentation/Usage.txt:169]:
```suggestion
The package Qt5 on brew is "keg-only", which means it is not "linked" with brew.
```

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.3-Ic-153.shtml

**Comment by sloriot** [comment]
@danston  Only 5.3?

**Comment by danston** [comment]
@sloriot We could also target 5.2.3, I guess.

**Comment by sloriot** [comment]
Note that we still maintain 5.1.x for now. Please rebase and update the branch base.

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.3-Ic-158.shtml

**Pull request merged**